runtime/compiler/aarch64/codegen/CallSnippet.cpp

```diff
@@ -432,11 +432,11 @@ uint8_t *TR::ARM64InterfaceCallSnippet::emitSnippetBody()
    *(int32_t *)cursor = methodSymRef->getCPIndexForVM();
    cursor += 4;
```

Comment:
I checked other platforms, and the cpIndex is expected to be in an 8-byte slot on 64-bits as well.

Reply:
Corrected the size not only this line but also in `ARM64UnresolvedCallSnippet` and `ARM64VirtualUnresolvedSnippet`.